client/app/containers/test-details/test-details.html

```diff
@@ -109,8 +109,9 @@
                             ng-repeat="btn in $ctrl.batchButtons"
                             flex="none"
                             class="{{  btn.class }}"
-                            ng-class="{ '_completed': btn.completed }"
-                            ng-click="btn.onClick($event,  btn)">
+                            ng-class="{ '_completed': btn.completed, '_disabled': !$ctrl.userHasAnyPermission(['MODIFY_TESTS']) }"
+                            ng-click="btn.onClick($event,  btn)"
+                            ng-disabled="!$ctrl.userHasAnyPermission(['MODIFY_TESTS'])">
```

Comment:
I think you don't need `_disabled` class, you can use valid `[disabled]` selector in the CSS

Reply:
fixed